### PR TITLE
SXT Babel: Better DX for invalid utility class name

### PR DESCRIPTION
**Pull request by michalsanger**
Screen Shot 2020-11-17 at 12 29 46

### CODE REVIEW

File: src/babel-plugin-transform-sx-tailwind/src/__tests__/__snapshots__/index.test.js.snap [hunk 1]
```diff
@@ -147,6 +147,23 @@ const __styles_3xR9dL = sx.create({
...
+import type { Node } from 'react';
+import { tailwind } from '@adeira/sx-tailwind';
+
+export default function Example(): Node {
+  return <div className={tailwind('bg-bleu-500')}>Button</div>;
+}
+
+      ↓ ↓ ↓ ↓ ↓ ↓
+
+Error: /Users/michalsanger/projects/adeira/universe/src/babel-plugin-transform-sx-tailwind/src/__tests__/fixtures/error/invalid-utility-class.js: Unknow utility name: bg-bleu-500
```
Comment (mrtnzlml) [inline src/babel-plugin-transform-sx-tailwind/src/__tests__/__snapshots__/index.test.js.snap:165]: This should be normalized to some path which will work for everyone.

**Review by mrtnzlml — CHANGES_REQUESTED** [review]

**Comment by mrtnzlml** [comment]
@michalsanger Is it related to this? https://github.com/adeira/universe/issues/1288 The error doesn't look that unusable. 🤔

File: src/babel-plugin-transform-sx-tailwind/src/__tests__/__snapshots__/index.test.js.snap (hunk 1 above)
Comment (michalsanger) [reply src/babel-plugin-transform-sx-tailwind/src/__tests__/__snapshots__/index.test.js.snap:165]: https://github.com/babel-utils/babel-plugin-tester/pull/80

**Comment by michalsanger** [comment]
Yes, related to that issue. Looks like Next is not swallowing Babel errors anymore.

**Review by mrtnzlml — APPROVED** [review]

**Pull request merged**